app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 28, [2, 2]),
+    (28, 29, [3, 3]),
+    (100, 100, [21, 17]),
+])
```

Comment:
add  ID and descriptive message  for clarity and ease of identifying test cases.